layouts/shortcodes/dashboard-table.html

```diff
@@ -0,0 +1,107 @@
+<script src="https://unpkg.com/tablesort@5.2.1/dist/tablesort.min.js"></script>
+<script src="https://unpkg.com/tablesort@5.2.1/dist/sorts/tablesort.number.min.js"></script>
+{{- with .Site.GetPage "/" -}}
+<table id="dashboard" class="sort">
+    <thead>
+        <tr>
+            <th>Section</th>
+            <th>Weight</th>
+            <th>State</th>
+            <th>Audit</th>
+            <th>Tests</th>
+        </tr>
+    </thead>
+    <tbody>
+        {{- template "dashboard-section-children" (dict "Section" . "level" 0) -}}
+    </tbody>
+</table>
+{{- end -}}
+<script>
+    new Tablesort(document.getElementById('dashboard')); 
+</script>
+{{ define "dashboard-section-children" }}
+    {{- $scratch := newScratch -}}
+    {{- $level := .level -}}
+    {{- $level = mul $level 10 -}}
+    {{- $pages := where .Section.Pages "Params.bookhidden" "ne" true -}}
+    {{- $pages2 := where $pages "Params.dashboardhidden" "ne" true -}}
+    {{- range sort $pages2 "Weight" "asc" -}}
+        {{- $level = add $level 1 -}}
+        {{- $scratch.Set "dashboard-weigth" 0 -}}
+        {{- $scratch.Set "dashboard-audit" 0 -}}
+        {{- if .IsSection -}}
+            <tr>
+                <td style="max-width:250px;white-space: nowrap;overflow: hidden;text-overflow: ellipsis;">{{- delimit (split $level "") "." }} <a title="{{.Title}}" href="{{.Permalink}}">{{.Title}}</a></td>
+                <!-- Weight -->
+                <td>{{- (default ($scratch.Get "dashboard-weigth") .Params.dashboardWeight) -}}</td>
+                <!-- State -->
+                {{ $color := "#6c8293"}}
+                {{- if eq .Params.dashboardState "incorrect" -}}
+                    {{ $color = "#ff0010" }}
+                {{- else if eq .Params.dashboardState "wip" -}}
+                    {{ $color = "#ff6f00" }}
+                {{- else if eq .Params.dashboardState "incomplete" -}}
+                    {{ $color = "#ffef00" }}
+                {{- else if eq .Params.dashboardState "stable" -}}
+                    {{ $color = "#90ff00" }}
+                {{- end -}}
+                <td style="background-color:{{- $color -}};color: black">{{- (default "N/A" (humanize .Params.dashboardState)) -}}</td>
```

Comment:
WIP is acronym, all caps 🧢 

```suggestion
                <td style="background-color:{{- $color -}};color: black">
                  {{- if eq .Params.dashboardState "wip" -}}
                    WIP
                  {{- else -}}
                    {{- (default "N/A" (humanize .Params.dashboardState)) -}}
                  {{- end -}}
                </td>
```